sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobAsyncClient.java

```diff
@@ -208,16 +218,20 @@ private SpecializedBlobClientBuilder prepareBuilder() {
      * <p>
      * Typically, the greater the number of buffers used, the greater the possible parallelism when transferring the
      * data. Larger buffers means we will have to stage fewer blocks and therefore require fewer IO operations. The
-     * trade-offs between these values are context-dependent, so some experimentation may be required to optimize inputs
+     * trade-offs between these values are context-dependent, so some experimentation may be required to optimize
+     * inputs
      * for a given scenario.
      *
      * <p><strong>Code Samples</strong></p>
      *
-     * {@codesnippet com.azure.storage.blob.BlobAsyncClient.uploadWithResponse#Flux-ParallelTransferOptions-BlobHttpHeaders-Map-AccessTier-BlobAccessConditions}
+     * {@codesnippet com.azure.storage.blob.BlobAsyncClient
```

Comment:
this will break codesnippets. same with the one below. they have an exception tob e longer than 120 characters

Reply:
IDE autoformatting. Thanks for catching.